Sources/NIOCore/EventLoop.swift

```diff
@@ -1107,6 +1107,22 @@ extension EventLoop {
         }
     }
 
+    /// Creates and returns a new isolated `EventLoopFuture` that is already marked as success. Notifications will be done using this `EventLoop` as execution `NIOThread`.
```

Comment:
```suggestion
    /// Creates and returns a new isolated `EventLoopFuture` that is already marked as success. Notifications will be done using this `EventLoop.
```

Reply:
NIOThread isn't public and also not all ELs are implemented using NIOThread

Reply:
oh, this seems to be in other places too. Probably some very old oversight ;)

Reply:
Yeah, it's a worthy doc comment change in the general case, but we can avoid making it again here.